code/__HELPERS/type2type.dm

```diff
@@ -424,4 +424,23 @@ proc/tg_text2list(text, glue=",", assocglue=";")
 		if(temp <= 16)
 			. = 0
 		else
-			. = max(0, min(255, 138.5177312231 * log(temp - 10) - 305.0447927307))
\ No newline at end of file
+			. = max(0, min(255, 138.5177312231 * log(temp - 10) - 305.0447927307))
+
+//Argument: Give this a space-separated string consisting of 6 numbers. Returns null if you don't
+/proc/text2matrix(var/matrixtext)

```

Comment:
The BYOND `matrix` proc accepts only 6 arguments - unless there's undocumented behavior I'm unaware of. This results in a matrix shaped as follows:

a d 0
b e 0
c f 1